src/inlineSuggestions/inlineSuggestionState.ts

```diff
@@ -55,3 +55,10 @@ export function getAllSuggestions(): ResultEntry[] | undefined {
 function results(): ResultEntry[] | undefined {
   return autocompleteResult?.results;
 }
+
+export function getHintColor(): string {
```

Comment:
why is it here ?

Reply:
thought it can be considered as part of the suggestions state - the color's state which is being pulled from the settings. Should i move this to a new file?